src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php

```diff
@@ -93,22 +150,89 @@ public function isRisky()
      */
     protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     {
-        $functionNames = $this->getFunctionNames();
+        $functionFilter = $this->getFunctionFilter();
 
-        $indexes = [];
+        if ('namespaced' === $this->configuration['scope']) {
```

Comment:
We could prevent `foreach` inside of `if` with changing `'namespaced'` to `'global'` here.

Reply:
:+1: done